conf/core/ConfigOptions.asciidoc

```diff
@@ -144,6 +144,34 @@ See also:
 
  * _Estimate Pairwise Relationships_, <<hootalgo>>
 
+=== changeset.buffer
+
+* Data Type: double
+* Default Value: `0.0`
+
+Value in degrees by which the specified AOI (convert.bounding.box) will be expanded when calculating
+changesets.  Setting this too large with some datasets may greatly increase the changeset
+derivation time.
+
+=== changeset.allow.deleting.from.features
+
+* Data Type: bool
+* Default Value: `true`
+
+If true, changesets derived can issue delete statements for the 'from' dataset (first dataset
+passed to the changeset deriver).  If false, no delete statements will be issued for the 'from'
+dataset.
```

Comment:
I have an idea for this that *might* work.  So far, all the features being dropped are new features created by conflation as a result of splitting a ref feature where the new features fall completely outside of the bounds (I think).  If that's the case, then I should be able to protect only those features and not all ref features.  Will try it.